angryexception/__init__.py,sha256=DcZSNexXIgqp4XAthqHIBUAwUIjZ_TeRs3LrtrKh58s,1018
angryexception-1.0.2.dist-info/WHEEL,sha256=y3eDiaFVSNTPbgzfNn0nYn5tEn1cX6WrdetDlQM4xWw,83
angryexception-1.0.2.dist-info/METADATA,sha256=X1bSe7eUD4O3O7Mt-O9_VKgRYflUPnwCNpY3lXYvMZM,921
angryexception-1.0.2.dist-info/RECORD,,
